autogen/agentchat/contrib/capabilities/transforms.py

```diff
@@ -108,33 +105,39 @@ class MessageTokenLimiter:
 
     The truncation process follows these steps in order:
 
-    1. Messages are processed in reverse order (newest to oldest).
-    2. Individual messages are truncated based on max_tokens_per_message. For multimodal messages containing both text
+    1. The minimum tokens threshold is checked (0 by default). If it is not reached, the messages are returned as is.
+        If it is reached, the following process is applied.
+    2. Messages are processed in reverse order (newest to oldest).
+    3. Individual messages are truncated based on max_tokens_per_message. For multimodal messages containing both text
         and other types of content, only the text content is truncated.
-    3. The overall conversation history is truncated based on the max_tokens limit. Once the accumulated token count
+    4. The overall conversation history is truncated based on the max_tokens limit. Once the accumulated token count
         exceeds this limit, the current message being processed get truncated to meet the total token count and any
         remaining messages get discarded.
-    4. The truncated conversation history is reconstructed by prepending the messages to a new list to preserve the
+    5. The truncated conversation history is reconstructed by prepending the messages to a new list to preserve the
         original message order.
     """
 
     def __init__(
         self,
-        max_tokens_per_message: Optional[int] = None,
-        max_tokens: Optional[int] = None,
+        max_message_tokens: Optional[int] = None,
+        max_history_tokens: Optional[int] = None,
```

Comment:
Not very clear what the variables are doing. Maybe `max_tokens_threshold_message`, `max_tokens_threshold_history` and `min_tokens_threshold`

Reply:
I just reevaluated my decisions, let's avoid renaming the existing arguments. It will cause bigger breaking change than I initially thought